src/main/io/gps.c

```diff
@@ -528,6 +529,21 @@ void gpsUpdate(timeUs_t currentTimeUs)
                 // remove GPS from capability
                 sensorsClear(SENSOR_GPS);
                 gpsSetState(GPS_LOST_COMMUNICATION);
+            } else {
+#if !defined(GPS_UBLOX_MODE_PEDESTRIAN)
```

Comment:
This can be moved up one line to include the `} else { as well.

Reply:
sure!